src/index.js

```diff
@@ -11,9 +11,15 @@ import { finalize, submit } from './report.js';
 
 		await submit(logger, context, inputs, report);
 
-		const { summary } = report;
+		const { summary: { githubOrganization, githubRepository } } = report;
+		const processedContext = {
+			github: {
+				organization: githubOrganization,
+				repository: githubRepository
+			}
+		};
```

Comment:
This is a holdover from the context being flat. and me now nesting it. Once the report context is nested this won't be needed. I may add a way to get some of the GitHub information from the report object to make this something that's mapped in the node package instead of the action.